### ISSUE TITLE
Decouple Chromecast Custom ID from Repo

### ISSUE THREAD

**Issue by jamescahall** [ConnectSDK/Connect-SDK-iOS#121]
Currently kCustomChromecastAppId is set in a define statement within the SDK.  This requires modification of the module in projects which causes issues with merge conflicts in getting the latest version of the repo.

This should be designed to pull the ID from the info.plist file for a project instead of using a define in code.  This will ensure no merge conflicts.

Thanks,
James

**Comment by eunikolsky** [comment]
@jamescahall, do you mean the `kGCKMediaDefaultReceiverApplicationID` constant we're using to play media? We can expose a property in the `CastService` interface to be able to set your own app ID if you need that.

**Comment by jamescahall** [comment]
Yes sorry.  I renamed this for my purposes in my code.  There should be an easy way to override this to use custom Chromecast application IDs.

I've had to do something like this:
    [self playMedia:mediaInformation webAppId:kCustomChromecastAppId success:success failure:failure];

With the define. Which replaces kGCKMediaDefaultReceiverApplicationID.  It seems this should be some sort of variable that can be configured OUTSIDE the SDK source as either a property or linked to plist data.
